viewer/src/dataModels/cad/internal/sector/WantedSector.ts

```diff
@@ -4,12 +4,10 @@
 
 import { SectorMetadata, SectorModelTransformation } from './types';
 import { LevelOfDetail } from './LevelOfDetail';
-import { ModelDataRetriever } from '@/utilities/networking/ModelDataRetriever';
 import { SectorScene } from './SectorScene';
 
 export interface WantedSector {
-  cadModelIdentifier: string;
-  dataRetriever: ModelDataRetriever;
+  blobUrl: string;
```

Comment:
I'm a bit unsure about `blobUrl` to identify model. We use this as a modelIdentifier in places where it doesn't make sense to know that `blobUrl` identifies a model. Also it seems that `WantedSector` should identify what to load, but `blobUrl` basically defines how to load the data...

Reply:
So, I have been thinking about this, and honestly, blob url makes so much more sense, since like we have said, the model being showed isn't the model, but a revision, stored in a single blob.

I have looked at this part of our code for a while, and well the most consistent way of dealing with the data loading is with blobUrl, and having a blobUrl Provider (this is also consistent with how PointClouds currently work).

What I would agree with, and this is how I would do it in the future, is to have the modelUrl cached, and then rather save the identifier which reacquires the blob url with the identifier, but I believe that to be a second step if anything.

Reply:
Are we sure that a model will be identifies by a single blob? Could it be that a model consists of two different outputs? An example I can think of is that a model might consist of 3D geometry + 2D "minimaps". These can probably be split into two different things, but are we sure this is the best way to do it? Remember that they are tightly linked, e.g. have some common coordinate system. I'm not sure blobUrl as an ID prohibits this to work well, I'm just not sure that it _doesn't_.

And, I think the argument about choosing an ID that is as generic as possible to hold...

I won't keep arguing to much about this, but I'm still not convinced that this is the right approach.